pyvo/mivot/utils/xml_utils.py

```diff
@@ -0,0 +1,80 @@
+"""
+Created on 16 Dec 2021
+
+@author: laurentmichel
+"""
+from lxml import etree
+from pyvo.mivot.utils.constant import Constant
+from pyvo.mivot.utils.vocabulary import Ele, Att
+from doctest import Example
```

Comment:
Don't use doctest this way, examples in docstrings and in narrative docs are tested with doctestplus, see the rest of the package for the particulars